utils/docker-network-ipaddresspool.sh

```diff
@@ -13,16 +13,30 @@ YQ="${2:-yq}"
 
 ## Parse kind network subnet
 ## Take only IPv4 subnets, exclude IPv6
-SUBNET=$(docker network inspect $networkName --format '{{json .IPAM.Config }}' | \
-    ${YQ} '.[] | select( .Subnet | test("^((25[0-5]|(2[0-4]|1\d|[1-9]|)\d)\.?\b){4}/\d+$")) | .Subnet')
+SUBNET=""
+
+# Try podman version of cmd first. docker alias may be used for podman, so network
+# command will be different
+set +e
+if command -v podman &> /dev/null; then
+  SUBNET=`podman network inspect -f '{{range .Subnets}}{{if eq (len .Subnet.IP) 4}}{{.Subnet}}{{end}}{{end}}' $networkName`
+fi
+set -e
+
+# Fallback to docker version of cmd
+if [[ -z "$SUBNET" ]]; then
+  SUBNET=`docker network inspect $networkName -f '{{ (index .IPAM.Config 0).Subnet }}'`
+fi
+
+# Neither worked, error out
 if [[ -z "$SUBNET" ]]; then
    echo "Error: parsing IPv4 network address for '$networkName' docker network"
    exit 1
 fi
 
 # shellcheck disable=SC2206
 subnetParts=(${SUBNET//./ })
-cidr="${subnetParts[0]}.${subnetParts[1]}.200.0/24"
+cidr="${subnetParts[0]}.${subnetParts[1]}.0.252/30"
```

Comment:
I'm not sure what implications this has. I assume it was changed for a reason? @eguzki I think you were the last one looking at this, do you think this will cause any issues?

Reply:
Reason for changing is to be compatible with podman and docker.

With docker, the default subnet is typically a /16 e.g. `172.17.0.0/16`
With podman, it's landing as a /24 for me locally:

```
podman network inspect kind|grep subnet
          "subnets": [
                    "subnet": "fc00:f853:ccd:e793::/64",
                    "subnet": "10.89.0.0/24",
```

Without this change, the .200.0/24 subnet wouldn't be part of the network subnet.
With the change, a .0.252/30 subnet would be on both podman & docker networks. (albeit with just 4 ip addresses for load balancers, which may be fine for most local dev use cases)

Reply:
sorry guys, I missed this. 

I came back to this PR because these changes are causing one more side effect https://github.com/Kuadrant/kuadrant-operator/pull/557